projects/arlas-toolkit/src/lib/components/register/register.component.html

```diff
@@ -4,7 +4,7 @@
       <div class="title">
         <h2>ARLAS - {{'Sign up' | translate}}</h2>
       </div>
-      <div>
+      <div *ngIf="allowRegister; else notAllowed">
         <div class="subtitle">
           {{'It’s quick and easy !' | translate}}
```

Comment:
There is no space before a ! in english and spanish => to change only in i18n file ?

Reply:
done